presto-hive/src/test/java/com/facebook/presto/hive/TestOrcPageSourceMemoryTracking.java

```diff
@@ -338,10 +338,10 @@ public void testTableScanOperator()
             assertFalse(operator.isFinished());
             Page page = operator.getOutput();
             assertNotNull(page);
-            page.getBlock(1);
+            page.getBlock(0);
             if (memoryUsage == -1) {
                 memoryUsage = driverContext.getSystemMemoryUsage();
-                assertBetweenInclusive(memoryUsage, 460000L, 469999L);
+                assertBetweenInclusive(memoryUsage, 360000L, 469999L);
```

Comment:
Instead of widening the range to include aria and non-aria cases, consider setting the range based on whether aria is enabled or not: 
```
if (isAriaScanEnabled(driverContext.getSession())) {
    assertBetweenInclusive(memoryUsage, 360000L, 369999L);
}
else {
    assertBetweenInclusive(memoryUsage, 460000L, 469999L);
}
```